libs/layout/src/lib/components/accordion/item/accordion-item.component.scss

```diff
@@ -0,0 +1,17 @@
+:host {
+	details > summary {
+		list-style: none;
+	}
+
+	details > summary::-webkit-details-marker {
```

Comment:
Should there not be any forced indication that this element is clickable / expandable? By removing this marker, this could very well be just a couple of plain text sentences. Personally, his does not feel very accessible.

Reply:
It's not mandatory according to the WCAG guidlines no. [And if you do, those should also be aria-hidden. ](https://www.a11y-collective.com/blog/accessible-accordion/) So in terms of accessibility there's no issue here, the details element already provides enough accessibility information for end-users.

The templates allow you to add the marker yourself, so we have to remove it here. Especially given that some designs will want it at the end or use a completely different icon.

Reply:
Yeah, that's fair reasoning. Would it be too opinionated to encourage users to provide an indicator in the documentation?

Reply:
I don't really think it's _really_ necessary, I think the vast majority of users will just use an indicator either way. If it was strongly suggested in terms of accessibility I would agree, but given that it's not, it's fair game to add it or not.